src/components/CoverPhoto/CoverPhoto.stories.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import {storiesOf} from "@storybook/react";
+import {withInfo} from "@storybook/addon-info";
+import {withKnobs, text} from "@storybook/addon-knobs";
+import CoverPhoto from "./index";
+
+storiesOf("CoverPhoto", module)
+    .addDecorator(
+        withInfo({
+            text: "This is a CoverPhoto component"
+        })
+    )
+
+    .addDecorator(withKnobs()
```

Comment:
@AnthoniaOkafor, can you remove this line of code? We shouldn't add a decorator if we're not using it.

Reply:
ok

Reply:
Hello @meekg33k . Could you review this PR again? The line of code ".addDecorator(withKnobs()" is being used.